services/packagist/packagist-dependency-version.tester.js

```diff
@@ -0,0 +1,88 @@
+import { isComposerVersion } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('gets the package version of symfony 5.2.3 - without query params')
+  .get('/symfony/symfony/v5.2.3.json')
+  .expectBadge({
+    label: 'dependency version',
+    message: 'version requirement not found',
+  })
```

Comment:
```suggestion
```

This test is a bit muddled and can be deleted.
What `/symfony/symfony/v5.2.3` is really doing is looking for a package that symfony requires which is called "v5.2.3" and then failing to find it, but the description and code don't really make that clear. You've got another test further down called "gets the package version of symfony 5.2.3 - incorrectly with missing part of dependency's name" which is testing the same thing really.

Reply:
deleted